docs/userGuide/reusingContents.md

```diff
@@ -257,6 +267,33 @@ In other words, **`<include>` interprets the reused code relative to the origina
 
 <hr><!-- ======================================================================================================= -->
 
+#### Specifying Variables in the Include Tag
+
+**It is possible to include variables in the include tag.**
+
+<div class="indented">
+
+{{ icon_example }} Specifying `title` and `author` variables in an `include` tag:
+
+```html
+<include src="article.md">
+  <span id="title">My Title</span>
+  <span id="author">John Doe</span>
+</include>
+```
+
+In `article.md`:
+
+<code>
+# {<span></span>{ title }}
+
+Author: {<span></span>{ author }}
+</code>
+</div>
+
+
+These variables work the same way as variables in `_markbind/variables.md`, except they only apply to the included file. This can be helpful for creating a template for multiple files.
```

Comment:
1. Fixing the grammar.
2. Rephrased the explanation of it being a template mechanism.

```markdown
..., except that they only apply to the included file. They allow the included file to be reused as a template, for different source files using different variable values.
```

Reply:
Side question: do they also work as nunjucks variables? e.g., `{{ (size | int) - 1 }}` would evaluate to 3 if `<span id="size">4</span>`

Reply:
Tested and it works - but do we want to document this? It seems like a small enough case, and it might confuse the user since it introduces another api

Reply:
We already claim that MarkBind is generally compatible with nunjucks syntax. Here, we are using nunjucks syntax for a MarkBind feature. It's better to clarify that these variables are in fact equivalent to nunjucks variables declared within the scope of the included file. However, it can be mentioned in https://markbind.github.io/markbind/userGuide/markBindSyntax.html#support-for-nunjucks rather than under this feature. What do you think?